### PR TITLE
Cyborg Rescuer Profession for Lab Start

### PR DESCRIPTION
**Pull request by LilimDiam**
#### Summary
Content "Adds a new profession "Cyborg Rescuer" to Lab Patient start"

#### Purpose of change
Provide a profession with tools and skills that offer player an opportunity to attempt and save every cyborg they encounter, along with alluding to existence of a group dedicated to freeing cyborgs.

#### Describe the solution
Created a new profession with skills in electronics, first aid, computers and hm12 dazzle rifle.

#### Describe alternatives you've considered
Waiting for Exodii lore to get completed and making profession fit into new lore, however I decided to lay down the groundwork now and apply changes to profession later.

#### Testing
Ran a lab start with new profession, all works as intended, character is (barely) able to disable cyborgs.

### CODE REVIEW

**Comment by SariusSkelrets** [comment]
Weren’t these cyborgs unknown from the masses pre-cataclysm?

Even in the old lore, the broken cyborgs were kept in labs conceived so those that knew could live inside to keep the secret with them. If someone knew about the broken cyborgs, the public outcry would’ve been impossible to ignore 

In the new lore, any bionic XEDRA found (if they found any) would’ve been treated like a treasure, installed into one of their troops or studied and reverse-engineered, not left to waste on some random individual filled to the brim with defective bionics

I guess the broken cyborgs could’ve been XEDRA’s attempts to reproduce and install bionics they found in the wild without the Exodii’s knowledge but then it would be so top secret the info being know enough for activist groups to exist sounds extremely unlikely 

Also the dazzler rifle is Hub 01 technology. I don’t know how they could’ve got one as the Hub is heavily guarded and self-sufficient

File: data/json/items/gun/ups.json [hunk 1]
```diff
@@ -152,6 +152,20 @@
     "ammo_effects": [ "LASER", "ROBOT_DAZZLE" ],
     "flags": [ "NEVER_JAMS", "NO_UNLOAD", "NON_FOULING", "NEEDS_NO_LUBE" ]
   },
+ {
+    "id": "makeshift_scrambler",
+    "type": "GUN",
+    "copy-from": "hm12",
+    "name": { "str": "Makeshift Scrambler Rifle" },
+    "description": "This 'energy weapon' appears to be haphazardly jury-rigged from commercially available electronics. It has a radar dish placed where a barrel would be, and the frame of a weapon is covered in a mesh of wiring. Despite its appearance, this weapon can temporarily stun robots if you manage to hit their sensors.",
```
Comment (Termineitor244) [inline data/json/items/gun/ups.json:160]:
```suggestion
    "description": "This 'energy weapon' appears to be haphazardly jury-rigged from commercially available electronics.  It has a radar dish placed where a barrel would be, and the frame of a weapon is covered in a mesh of wiring.  Despite its appearance, this weapon can temporarily stun robots if you manage to hit their sensors.",
```

File: data/json/items/gun/ups.json [hunk 2]
```diff
@@ -152,6 +152,20 @@
     "ammo_effects": [ "LASER", "ROBOT_DAZZLE" ],
     "flags": [ "NEVER_JAMS", "NO_UNLOAD", "NON_FOULING", "NEEDS_NO_LUBE" ]
   },
+ {
```
Comment (Termineitor244) [inline data/json/items/gun/ups.json:155]:
```suggestion
  {
```

File: data/mods/package_bionic_professions/bionic_professions.json [hunk 3]
```diff
@@ -763,5 +763,22 @@
...
+    "skills": [ { "level": 2, "name": "gun" }, { "level": 2, "name": "rifle" }, { "level": 2, "name": "firstaid" }, { "level": 2, "name": "computer" }, { "level": 3, "name": "electronics" } ],
+    "items": {
+      "both": {
+        "items": [ "jumpsuit", "hazmat_suit", "mask_gas", "socks", "molle_pack", "wristwatch" ],
+        "entries": [ { "group": "charged_smart_phone" }, { "item": "UPS_off", "ammo-item": "heavy_battery_cell", "charges": 1250 }, { "item": "makeshift_scrambler" } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    },
+"flags": [ "SCEN_ONLY" ]
```
Comment (Termineitor244) [inline data/mods/package_bionic_professions/bionic_professions.json:782]:
```suggestion
  },
  {
    "type": "profession",
    "id": "cyborg_rescue",
    "name": "Cyborg Rescuer",
    "description": "Before the cataclysm struck, you were one of the members of an underground group striving to free cyborgs from their shackles, raiding laboratories and underground clinics in your pursuit of liberation.  Now that most of the humanity has been turned into zombies, your goal has become even more important to you.",
    "points": 4,
    "skills": [
      { "level": 2, "name": "gun" },
      { "level": 2, "name": "rifle" },
      { "level": 2, "name": "firstaid" },
      { "level": 2, "name": "computer" },
      { "level": 3, "name": "electronics" }
    ],
    "items": {
      "both": {
        "items": [ "jumpsuit", "hazmat_suit", "mask_gas", "socks", "molle_pack", "wristwatch" ],
        "entries": [
          { "group": "charged_smart_phone" },
          { "item": "UPS_off", "ammo-item": "heavy_battery_cell", "charges": 1250 },
          { "item": "makeshift_scrambler" }
        ]
      },
      "male": [ "boxer_shorts" ],
      "female": [ "bra", "panties" ]
    },
    "flags": [ "SCEN_ONLY" ]
```

**Review by Termineitor244 — CHANGES_REQUESTED** [review]
You need to lint your changes (Using https://dev.narc.ro/cataclysm/format.html ) and to put double spaces after every non final point.

I linted the profession and added spaces where appropriate.

File: data/json/items/gun/ups.json (hunk 1 above)
Comment (LilimDiam) [reply data/json/items/gun/ups.json:160]: Oh, I see, will keep that in mind next time

File: data/json/items/gun/ups.json [hunk 4]
```diff
@@ -152,6 +152,20 @@
     "ammo_effects": [ "LASER", "ROBOT_DAZZLE" ],
     "flags": [ "NEVER_JAMS", "NO_UNLOAD", "NON_FOULING", "NEEDS_NO_LUBE" ]
   },
+  {
+    "id": "makeshift_scrambler",
+    "type": "GUN",
+    "copy-from": "hm12",
+    "name": { "str": "Makeshift Scrambler Rifle" },
+    "description": "This 'energy weapon' appears to be haphazardly jury-rigged from commercially available electronics.  It has a radar dish placed where a barrel would be, and the frame of a weapon is covered in a mesh of wiring.  Despite its appearance, this weapon can temporarily stun robots if you manage to hit their sensors.",
```
Comment (kevingranade) [inline data/json/items/gun/ups.json:160]: Big nope here, we don't do this kind of scifi.

**Comment by kevingranade** [comment]
This really doesn't fit anywhere in the lore, even in the mod, because even with cyborgs *existing* pre-cataclysm in enough numbers to be playable, organizations dedicated to sending rescue missions after them in particular *that just so happen to have exactly the tools the player needs* are beyond credulity.

File: data/json/items/gun/ups.json (hunk 4 above)
Comment (LilimDiam) [reply data/json/items/gun/ups.json:160]: Aw, shame. I assume the fact that it's meant for Cyborg professions mod doesn't change it?

Comment (LilimDiam) [reply data/json/items/gun/ups.json:160]: Nvm, saw your second comment, fair enough

**Pull request closed without merging**